content/help/ops/security/debugging-authorization/index.md

```diff
@@ -0,0 +1,257 @@
+    2018-07-26T22:25:41.183281Z debug rbac matched AccessRule[0]
+    2018-07-26T22:25:41.183390Z debug rbac generated filter config: {policies:<key:"productpage-viewer" value:<permissions:<and_rules:<rules:<or_rules:<rules:<header:<name:":method" exact_match:"GET" > > > > > > principals:<and_ids:<ids:<any:true > > > > >  }
+    2018-07-26T22:25:41.184407Z info  rbac built filter config for productpage.default.svc.cluster.local
+    {{< /text >}}
+
+    It means Pilot have generated:
+
+    * An empty config for `sleep.foo.svc.cluster.local` as there is no authorization policies matched
+      and Istio will deny all requests sent to this service by default.
+
+    * An config for `productpage.default.svc.cluster.local` and Istio will allow anyone to access it
+      with GET method.
+
+## Ensure Pilot Distributes Policies to Proxies Correctly
+
+Pilot distributes the authorization policies to proxies. The following steps help you ensure Pilot
+is working as expected:
+
+> Note: The command used in this section assumes you have deployed [Bookinfo application](/docs/examples/bookinfo/),
+otherwise you should replace `"-l app=productpage"` with your actual pod.
+
+1. Run the following command to get the proxy configuration dump for the `productpage` service:
+
+    {{< text bash >}}
+    $ kubectl exec  $(kubectl get pods -l app=productpage -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- curl localhost:15000/config_dump -s
+    {{< /text >}}
+
+1. Check the log and verify:
+
+    * It includes a `envoy.filters.http.rbac` filter that is used to enforce the authorization policy
+      on each incoming request.
+    * The filter gets updated accordingly after you updated your authorization policy.
+
+1. For example, you might see something similar to the following:
+
+    {{< text plain >}}
+    {
+     "name": "envoy.filters.http.rbac",
+     "config": {
+      "rules": {
+       "policies": {
+        "productpage-viewer": {
+         "permissions": [
+          {
+           "and_rules": {
+            "rules": [
+             {
+              "or_rules": {
+               "rules": [
+                {
+                 "header": {
+                  "exact_match": "GET",
+                  "name": ":method"
+                 }
+                }
+               ]
+              }
+             }
+            ]
+           }
+          }
+         ],
+         "principals": [
+          {
+           "and_ids": {
+            "ids": [
+             {
+              "any": true
+             }
+            ]
+           }
+          }
+         ]
+        }
+       }
+      },
+      "shadow_rules": {
+       "policies": {}
+      }
+     }
+    },
+    {{< /text >}}
+
+    It means the proxy of productpage has enabled the `envoy.filters.http.rbac` filter with rules
+    that allows anyone to access it via GET method. The `shadow_rules` is not used and could be ignored safely.
```

Comment:
that allow anyone to access it via `GET` method. The `shadow_rules` are not used and you can ignore them safely.